internal/git/rebase.go

```diff
@@ -1,12 +1,18 @@
 package git
 
 import (
+	"fmt"
+	"os"
+	"path/filepath"
 	"regexp"
 	"strings"
 
 	"github.com/sirupsen/logrus"
 )
 
+// ref: https://git-scm.com/docs/git-rev-parse#Documentation/git-rev-parse.txt-codeREBASEHEADcode
+const REBASE_STATE_FILE = "REBASE_HEAD"
```

Comment:
In other places, `REBASE_HEAD` is defined as string directly. 
Should I align with that here as well? It means that I shouldn't define this string as const?